packages/type-extrinsics/test/e2e/transfer.spec.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017-2019 @polkadot/extrinsics authors & contributors
```

Comment:
ooh I forgot we had e2e tests in type-extrinsics. @jacogr I propose to remove this one, and only put them in api

Reply:
Agreed. Nobody knows about them here, so they never get used.

Reply:
Ah, dammit, they're failing anyways...

Reply:
The tests are broken in the current master, too, so they didn't break as part of this PR and are unrelated.
Therefore I added two `@TODO`s to code:

```
// @TODO 1: All three tests below are broken, fix or find better way to test transfers.
// @TODO 2: Move tests out of ./type-extrinsics to ./api folder
```